src/Features/Core/Portable/ImplementInterface/AbstractImplementInterfaceService.CodeAction_Property.cs

```diff
@@ -69,6 +51,68 @@ internal partial class ImplementInterfaceCodeAction
                     setMethod: setAccessor);
             }
 
+            /// <summary>
+            /// Lists compiler attributes that we want to remove.
+            /// The TupleElementNames attribute is compiler generated and used for naming tuple element names. We never want to place it in source code.
```

Comment:
you don't have to fix this.  But generally we would like all code to fit within github's width.  Makes PRs easier.  Thanks!